src/precompiles/ArbNativeTokenManager.sol

```diff
@@ -11,6 +11,16 @@ pragma solidity >=0.4.21 <0.9.0;
  *         Available in ArbOS version 41
  */
 interface ArbNativeTokenManager {
+    /**
+     * @notice Emitted when some amount of the native gas token is minted by a NativeTokenOwner.
+     */
+    event NativeTokenMinted(address indexed minter, address indexed to, uint256 amount);
```

Comment:
Won't minter and to always be the same address?

Reply:
oh right will fix